client/src/presentation/3-organisms/error-screen.tsx

```diff
@@ -0,0 +1,47 @@
+import { UserDataApi } from "../../api/user-data";
+import React, { MouseEvent } from "react";
+import { FC } from "react";
+import { ContinueButton } from "../1-atoms/continue-button";
+import { QuestionBar } from "../2-molecules/question-bar";
+import { useUserDataDispatch, useUserDataState } from "../../context/context";
+import { useNavigate } from "react-router-dom";
+import { setUserUuid } from "../../context/actions";
+import { useState } from "react";
+import { UserGameResponse } from "server";
+
+interface Props {
+  //
+}
+
+export const ErrorScreen: FC<Props> = () => {
+  const context = useUserDataState();
+  const userDataDispatch = useUserDataDispatch;
+  const navigate = useNavigate();
+  const [showSnackbar, setShowSnackbar] = useState<boolean>(false);
+  const restartGame = (e: MouseEvent) => {
+    UserDataApi.resetGame(context.uuid)
+      .then((res: UserGameResponse) => {
+        setUserUuid(res.uuid, userDataDispatch).then(() => {
+          navigate(`/game/${res.round}`);
+        });
+      })
+      .catch((err) => {
+        console.log(err);
+        setShowSnackbar(true);
+      });
+  };
+
```

Comment:
you can async/await this :) 
```suggestion
  const restartGame = async (e: MouseEvent) => {
    try {
        const response = await UserDataApi.resetGame(context.uuid);
        navigate(`/game/${res.round}`);
    } catch (error) {
      console.error(err);
      setShowSnackbar(true);
    }
  };

```